DataFormats/Headers/include/Headers/DataHeader.h

```diff
@@ -215,33 +201,41 @@ constexpr T String2(const char (&str)[N])
 /// have to be studied.
 template <int N, typename PrinterPolicy = Internal::defaultPrinter>
 struct Descriptor {
-  static_assert(Internal::NumberOfActiveBits<N>::value == 1 && N <= 8,
+  static_assert(Internal::NumberOfActiveBits<N>::value == 1 && !(N%2),
```

Comment:
not sure what to do here, maybe we want to enforce N to be a multiple of the size of the smallest supported ItgType? Or something similar?